content/pt/docs/3.features/10.transitions.md

```diff
@@ -1,49 +1,49 @@
 ---
-title: Transitions
-description: Nuxt uses the transition component to let you create amazing transitions/animations between your routes.
+title: Transições
+description: O Nuxt usa o componente de transição para permitir que você criar animações e transições incríveis entre suas rotas.
 category: features
 csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/03_features/05_transitions?fontsize=14&hidenavigation=1&theme=dark
 ---
-# Transitions
+# Transições
 
-Nuxt uses the [transition component](http://vuejs.org/v2/guide/transitions.html#Transitioning-Single-Elements-Components) to let you create amazing transitions/animations between your routes.
+O Nuxt usa o [componente de transição](http://vuejs.org/v2/guide/transitions.html#Transitioning-Single-Elements-Components) para permitir que você criar animações e transições incríveis entre suas rotas.
 
 ---
 
-To define a custom transition for a specific route add the `transition` key to the page component.
+Para definir uma transição personalizada para uma rota específica, adiciona a chave `transition`  ao componente da página.
 
 ```js{}[pages/index.vue]
 export default {
-  // Can be a String
+  // Pode ser uma String
   transition: ''
-  // Or an Object
+  // Ou um Objeto
   transition: {}
-  // or a Function
+  // Ou uma Função
   transition (to, from) {}
 }
 ```
 
-## String
+## Sequência de Caracteres
```

Comment:
Here `String` is translated but it's not above. I'm not sure it needs to be

Reply:
👌 
Don't take the french version too serisously 😅  it may have some typos. Even if Nuxt is french, the english version stays the most contributed one!